uint32 id
uint8 type
uint8 blockSize
Block[] blockList
